include/BALL/VIEW/RENDERING/RENDERERS/glRenderer.h

```diff
@@ -38,13 +38,13 @@
 #endif
 
 #ifdef BALL_OS_DARWIN
+        #include <OpenGL/gl.h>
 	#include <OpenGL/glu.h>
 #else
+        #include <GL/gl.h>
 	#include <GL/glu.h>
 #endif
 
-#include <QtGui/QOpenGLFunctions_1_3>
```

Comment:
I am not 100% sure, but the problem why QOpenGLFunctions does not work may be that glu.h implicitly includes gl.h. We only use GLU for creating spheres, disks and cylinders, but getting rid of it would require to write quite some boilerplate code...